lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -393,6 +398,10 @@ module InternalConstants
     config_param :monitoring_type, :string,
                  :default => Monitoring::PrometheusMonitoringRegistry.name
 
+    # Whether to autoformat value of "logging.googleapis.com/trace" to
```

Comment:
It would be nice to provide more informative description on what this configuration does to help guide users. How about:
```
Whether to autoformat value of "logging.googleapis.com/trace" to comply with Stackdriver Trace format ("projects/[PROJECT-ID]/traces/[TRACE-ID]") when setting the trace field of the LogEntry object.
```

BTW, once this is released, we can update the [public documentation](https://cloud.google.com/logging/docs/agent/configuration#cloud-fluentd-config) to add this new config.

Reply:
updated the comment to clarify the config!